src/config/index.ts

```diff
@@ -0,0 +1 @@
+export * from './config';
```

Comment:
it's no need to add index.ts file to each directory, it should be added only to root folder(module)

Reply:
why? 
this makes it easier to import => `import { config } from '../../config';`